packages/daimo-api/src/server/cron.ts

```diff
@@ -111,12 +112,30 @@ export class Crontab {
       address: faucetAddr,
     });
     const balanceEth = Number(formatEther(balance));
+    console.log(`[CRON] checked faucet ETH balance ${balanceEth}`);
+
     await this.sendLowBalanceMessage(
       balanceEth,
       `Faucet ${faucetAddr} ETH`,
       0.05,
       0.005
     );
+
+    const balanceUSDC = await this.vc.publicClient.readContract({
+      abi: erc20ABI,
+      address: chainConfig.tokenAddress,
+      functionName: "balanceOf",
+      args: [faucetAddr],
+    });
+    const balanceDollars = Number(amountToDollars(balanceUSDC));
+    console.log(`[CRON] checked faucet USDC balance ${balanceDollars}`);
+
+    await this.sendLowBalanceMessage(
+      balanceDollars,
+      `Faucet ${faucetAddr} USDC`,
+      250,
+      25
```

Comment:
recommend
```
isTestnet ? 20 : 250,
isTestnet ? 5 : 25
```
... since we are low on base sepolia usdc

Reply:
hm I think it's kind of good to get the cron / notifs to match closely since those alerts go into a separate slack channel anyway. will leave as is unless you feel strongly.